python/buckets.cpp

```diff
@@ -86,6 +86,18 @@ void init_buckets(py::module &m) {
         return dataset::buckets::concatenate(a, b);
       },
       py::call_guard<py::gil_scoped_release>());
+  buckets.def(
+      "concatenate",
+      [](const VariableConstView &var, const Dim dim) {
+        return dataset::buckets::concatenate(var, dim);
+      },
```

Comment:
Would be nice with some `py::arg` and docstrings here, but i see that there is none in the rest of the file, so can be done in a later step.